plugins/modules/vmware_drs_override.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
#!/usr/bin/python
# -*- coding: utf-8 -*-

# Copyright: (c) 2024, Sergey Goncharov <your@email.address>
# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
# SPDX-License-Identifier: GPL-3.0-or-later
```

It looks like we're missing a a reference to the GPL 3 license here. Please add it (plus your e-mail address so you're properly mentioned and get the credit that you're entitled to).